person.rb

```diff
@@ -1,24 +1,14 @@
-# This file defines the Person class, representing individuals associated with the school library app.
-class Person
-  attr_accessor :name, :age
-  attr_reader :id
+# person.rb
+class Person < Nameable
+  attr_reader :name, :age
 
-  def initialize(age:, parent_permission: true, name: 'Unknown')
-    @id = Random.rand(1..1000)
-    @name = name
+  def initialize(age, name)
+    super()
     @age = age
-    @parent_permission = parent_permission
-  end
-
-  private
-
-  def of_age?
-    @age >= 18
+    @name = name
```

Comment:
- [ ] 🎉 Awesome job on the pull request! I noticed there was an error when executing your code.

![image](https://github.com/alexmedinasf/OOP_School_library/assets/98453979/5c024493-e22f-4b96-b941-23cad1011de8)

 - [ ] To address the uninitialized constant Nameable issue, could you please include `require_relative 'nameable'` This will ensure proper inheritance for the Person class. Keep up the great work! 🚀

### example 

```
# person.rb
require_relative 'nameable'

class Person < Nameable
  attr_reader :name, :age

  def initialize(age, name)
    super()
    @age = age
    @name = name
  end

  def correct_name
    name
  end
end

```